src/main/java/lab3/game/Board.java

```diff
@@ -0,0 +1,74 @@
+package lab3.game;
+
+public class Board {
+    private char[][] board; // 3x3 board to store 'X', 'O', or ' ' (empty)
+    private static final int SIZE = 3;
+
+    // Constructor to initialize an empty board
+    public Board() {
+        board = new char[SIZE][SIZE];
+        for (int i = 0; i < SIZE; i++) {
+            for (int j = 0; j < SIZE; j++) {
+                board[i][j] = ' ';
+            }
+        }
+    }
+
+    // Place a mark (X or O) at a specific position
+    public boolean placeMark(Position pos, char mark) {
```

Comment:
Same here

Reply:
Also, don't forget to document your pre and post conditions.  E.g. If the position is already taken, returns false.